components/shared/LanguageSwitcher/LanguageSwitchBtn.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { CheckIcon, ChevronDownIcon } from '@heroicons/react/solid';
+
+const LanguageSwitchBtn = () => {
+  const { locale } = useRouter();
+  const [languageModal, setLanguageModal] = useState<boolean>(false);
+  const [currentLanguage, setCurrentLanguage] = useState(locale);
+
+  const openLanChangeHandler = () => {
+    setLanguageModal((prevState) => !prevState);
+  };
+
```

Comment:
one jsx componenet per folder, u should make two folders called LanguageSwitchBtn and languageSwitchBTnss